packages/stylelint-plugin/tests/fixtures/scss.valid.scss

```diff
@@ -1,3 +1,4 @@
+// CSS nesting trigers a parse error if we're not using a customSyntax
```

Comment:
This is valid, do we need a comment?

Reply:
I think it fulfills the same purpose as the comments in invalid.scss - it says what's bit of code is important for this test output to pass.

- In invalid.scss tests will fail because expected stylelint output was not present if you remove the scss rules from the config 
- In valid.scss tests will fail because we expected stylelint output to be empty if you remove the customSyntax from the config